TS compilation issue

When I run `tsc` in achingbrain's `uint8arrays` repo I get:

```
node_modules/web-encoding/src/lib.d.ts:1:10 - error TS2661: Cannot export 'TextEncoder'. Only local declarations can be exported from a module.

1 export { TextEncoder, TextDecoder }
           ~~~~~~~~~~~

node_modules/web-encoding/src/lib.d.ts:1:23 - error TS2661: Cannot export 'TextDecoder'. Only local declarations can be exported from a module.

1 export { TextEncoder, TextDecoder }
                        ~~~~~~~~~~~
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,1 @@
+node_modules

--- a/src/lib.d.ts
+++ b/src/lib.d.ts
@@ -1,1 +1,4 @@
-export { TextEncoder, TextDecoder }
+type Encoder = TextEncoder
+type Decoder = TextDecoder
+
+export { Encoder as TextEncoder, Decoder as TextDecoder }

